### PR TITLE
Add merchants to list

### LINKED ISSUE #472: Add gameofcryptos.com
The owner contacted me. They actually can accept monero via third-party. Should we add to merchants directory?
http://gameofcryptos.com/

### LINKED ISSUE #473: Request for merchants & services page
We would like to add WalletMonero to your services list on the page: https://getmonero.org/community/merchants/

Previously we introduced our service to Monero community, which resulted in good reviews: https://www.reddit.com/r/Monero/comments/78nvn6/walletmonero_new_monero_light_wallet/

Kindly use the following link: https://walletmonero.com?utm_source=getmonero_org&utm_medium=list&utm_content=monero_list&utm_campaign=getmonero_org

In case of any questions don't hesitate to contact me: support@walletmonero.com

### LINKED ISSUE #483: Request for merchants & exchanges
We would like to add ChangeNOW to your exchanges list on the page: https://getmonero.org/community/merchants/

Kindly use the following link: https://changenow.io?utm_source=monero&utm_medium=link&utm_term=merchants&utm_campaign=monero

In case of any questions don't hesitate to contact me: info@changenow.io

### LINKED ISSUE #485: Request to add our business on Merchants & Services page
Hi, our business is now officially accepting Monero as a payment method. Could you please add us to the page "Merchants & Services" page?
Thank you very much!

### PR DESCRIPTION
**Pull request by erciccione**
~~Will upload with new merchants as soon as they are [submitted on Reddit](https://www.reddit.com/r/Monero/comments/79w8i7/list_of_servicesmerchantsplatforms_who_accept/).~~

Closes #472 
Closes #473 
Closes #483 
Closes #485

### CODE REVIEW

**Comment by MegaChange** [comment]
pls change url to https://www.megachange.is

**Comment by mattcode55** [comment]
I don't see Monero listed anywhere on the site.

**Comment by MegaChange** [comment]
Monero will be added in 1-2 weeks, our developers in the process of implementing it now

We need to get listed before or at the same time when monero become available for trading on our website.

**Comment by mattcode55** [comment]
>We need to get listed before or at the same time

I think it would be confusing to users if it was added before.

**Comment by MegaChange** [comment]
we don't think so
as monero will be added shortly as i mentioned

**Comment by erciccione** [comment]
I agree, listing this service now would be misleading. referencing [#466 comment](https://github.com/monero-project/monero-site/issues/466#issuecomment-341674813)

File: _data/merchants.yml [hunk 1]
```diff
@@ -191,6 +199,8 @@
       url: http://directvoltage.com
     - name: Fine Art from Jeanine King ~ Artwork of Home/Archtecture, Pets, Potraits, Caricatures ~ International Shipping
       url: http://art2unlimited.webs.com/
+    - name: Game of Cryptos
```
Comment (mattcode55) [inline _data/merchants.yml]: >We accept all major debit and credit cards through PayPal and over next days we will accept currencies like Ethereum and Litecoin via CoinPayments.
>
>Now only  accept Bitcoin.
>[https://gameofcryptos.com/apps/help-center#what-payment-methods-are-accepted](https://gameofcryptos.com/apps/help-center#what-payment-methods-are-accepted)

**Comment by ChangeNOW-Exchange** [comment]
Hello,

Could you please explain how long it will take to add the ChangeNow exchanger to the list?

**Comment by erciccione** [comment]
@ChangeNOW-Exchange This PR got a bit delayed since there were a lot of requests from various merchants during last month. Btw I'm not going to add any new one.

@luigi1111  this is good to go

**Comment by ChangeNOW-Exchange** [comment]
Thank you for your answers.

When are you planning to merge this pull requests?

**Comment by erciccione** [comment]
Since this PR was open for a while, I don't think it will take long (I'm not the one who merge btw)

**Comment by ChangeNOW-Exchange** [comment]
Thanks for the info, we will be expected.

**Comment by MegaChange** [comment]
please change the title from MegaChange.is -> MegaChange.is (P2P exchange)
It is important people to know that it is p2p trading

**Pull request merged**